PhysicsTools/NanoAOD/python/triggerObjects_cff.py

```diff
@@ -216,29 +216,44 @@
   selections = selections2016
 )
 
-from PhysicsTools.PatUtils.L1ECALPrefiringWeightProducer_cff import prefiringweight
+from PhysicsTools.PatUtils.L1PrefiringWeightProducer_cff import prefiringweight
 #Next line will be updated once we get UL2016 maps
-run2_jme_2016.toModify( prefiringweight, DataEra = cms.string("2016BtoH"))
+(run2_muon_2016 & tracker_apv_vfp30_2016).toModify( prefiringweight, DataEraECAL = cms.string("2016BtoH"),  DataEraMuon = cms.string("2016preVFP"))
+(run2_muon_2016 & ~tracker_apv_vfp30_2016).toModify( prefiringweight, DataEraECAL = cms.string("2016BtoH"),  DataEraMuon = cms.string("2016postVFP"))
```

Comment:
@lathomas I assume you want to switch to the UL2016 maps here, too?

Reply:
@JanFSchulte  Indeed !